cylance_protect/help.md

```diff
@@ -64,14 +113,9 @@ Example input:
 
 ```
 {
-  "add_zones": [
```

Comment:
Were these supposed to be removed?

Reply:
Yes! It seems like the last update to this plugin had an input example that did match the spec of the action...this was generated from regenerate